src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -45,6 +46,25 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
   using dt = dnnl::memory::data_type;
 
  public:
+  std::map<std::string, tag> layout_dict {
+    {"NCHW", tag::abcd},
+    {"OIHW", tag::abcd},
+    // preferred format of oneDNN v1.x on AVX512
```

Comment:
this reminds me that should we need to cover/test on pre-AVX512 platforms (SSE/AVX/AVX2)?
@ZhennanQin